flyway-core/src/test/resources/org/flywaydb/core/internal/dbsupport/saphana/procedures/procedureWithNestedBlocks.sql

```diff
@@ -0,0 +1,23 @@
+-- just to have one statement before the actual test statement
+CREATE VIEW all_misters AS SELECT * FROM test_user WHERE name LIKE 'Mr.%';
+
+-- a procedure with nested begin-end; blocks
+-- also taken from
+-- SAP HANA SQLScript Reference - SAP Library at:
+-- http://help-legacy.sap.com/saphelp_hanaplatform/helpdata/en/f0/a6dceac8b94cca98dd2741ac6541b8/content.htm?frameset=/en/c6/558a64245942ebb52f6a6ddb3e4278/frameset.htm
```

Comment:
Sorry, I used the "share" link on the SAP Help page, which seems not to include the full link.
I updated the link in file.